src/components/PlayerCardContainer/PlayerCardForm.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import './PlayerCardForm.css'
+import CustomButton from "./CustomButton";
+
+function PlayerCardForm(props) {
+    return (
+        <div className={'player-card-display'}>
+            <label className={'player-name-display'}>{props.player.name}</label>
```

Comment:
While I admire you trying to use a good semantic element for these display fields I think it might make sense to pick a different one.  <label> is a special HTML element that is intended to be used as a caption for a "labelable" element, such as a text input, button, or checkbox.  

[Label Element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)
[Text elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element#text_content)

Reply:
Thank you did not know about this. Ended up using a paragraph element, what do you think about that?